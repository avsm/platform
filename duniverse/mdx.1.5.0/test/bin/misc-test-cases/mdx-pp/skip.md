## Skipped blocks

```ocaml
let () = assert true
```

This block should not be executed by the tests:
```ocaml skip
let () = assert false (* Don't print this! *)
```

```ocaml
let () = assert true
```
