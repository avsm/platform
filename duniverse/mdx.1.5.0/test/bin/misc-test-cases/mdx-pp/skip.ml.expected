#4 "skip.md"
let () = assert true
;;
#13 "skip.md"
let () = assert true
;;
